src/java/org/apache/cassandra/utils/bytecomparable/ByteSource.java

```diff
@@ -766,19 +766,19 @@ static <V> ByteSource optionalFixedLength(ValueAccessor<V> accessor, V data)
      * underlying type has a fixed length.
      * In tests, this method is also used to generate non-escaped test cases.
      */
-    public static <V> ByteSource fixedLength(ValueAccessor<V> accessor, V data)
+    public static <V> ByteSource preencoded(ValueAccessor<V> accessor, V data)
```

Comment:
Nit: I might amend the javadoc a bit, in the sense that afaict, this is used in non-fixed length cases outside of tests too, in case where the input is already encoded (I guess the name makes this more clear now, but the javadoc still seems to suggest that the only non-test correct use case is fixed length).